### PR TITLE
Pitch and yaw support (/spawnnpc and /spawnplayer)

### PR DESCRIPTION
**Pull request by RobinMCNetwork**
I have not done this in an IDE, so there might be small mistakes.

### CODE REVIEW

File: Plugin/src/main/java/org/inventivetalent/npclib/command/SpawnCommands.java [hunk 1]
```diff
@@ -55,19 +55,19 @@ public void spawnNPC(CommandSender sender, String typeString, @OptionalArg Doubl
...
 			 errorHandler = FeedbackErrorHandler.class)
 	@Permission("npclib.command.spawnplayer")
-	public void spawnPlayer(CommandSender sender, String name, @OptionalArg String skin, @OptionalArg Double x, @OptionalArg Double y, @OptionalArg Double z, @OptionalArg String worldName) {
+	public void spawnPlayer(CommandSender sender, String name, @OptionalArg String skin, @OptionalArg Double x, @OptionalArg Double y, @OptionalArg Double z, @OptionalArg Double pitch, @OptionalArg Double yaw, @OptionalArg String worldName) {
 		if (name.length() > 16) {
 			sender.sendMessage("§cName is too long (" + name.length() + ">16)");
 			return;
 		}
-		Location location = parseLocation(sender, x, y, z, worldName);
+		Location location = parseLocation(sender, x, y, z, pitch, yaw worldName);
```
Comment (SirFaizdat) [inline Plugin/src/main/java/org/inventivetalent/npclib/command/SpawnCommands.java]: comma between yaw and worldname

File: Plugin/src/main/java/org/inventivetalent/npclib/command/SpawnCommands.java [hunk 2]
```diff
@@ -55,19 +55,19 @@ public void spawnNPC(CommandSender sender, String typeString, @OptionalArg Doubl
 
 	@Command(name = "spawnPlayer",
 			 aliases = { "spawnNpcPlayer" },
-			 usage = "<Name> [skin] [x] [y] [z] [world]",
+			 usage = "<Name> [skin] [x] [y] [z] [pitch] [yaw] [world]",
 			 description = "Spawn a Player NPC at your location or the specified coordinates",
 			 min = 1,
 			 max = 6,
```
Comment (InventivetalentDev) [inline Plugin/src/main/java/org/inventivetalent/npclib/command/SpawnCommands.java]: 'max' argument length needs to be updated as well.

**Comment by RobinMCNetwork** [comment]
Bumb

(I do really need it ASAP, and I don't really want to compile my fork :P)

**Comment by RobinMCNetwork** [comment]
Please?

**Comment by RobinMCNetwork** [comment]
I am sorry if I am really bothering you, but this is the only thing holding me back from finishing my server right now. I tried compiling the plugin myself, which did not work since not the complete code is on GitHub. It would be a real shame if the quest NPCs had to be villagers. If there's something I can do please tell me, I will be happy to do the work.

**Comment by DziNeIT** [comment]
might be worth tagging @InventivetalentDev in case her notification setting is off for issues

**Comment by RobinMCNetwork** [comment]
Thanks!!

**Pull request merged**